react-client/src/index.jsx

```diff
@@ -24,7 +24,7 @@ class App extends React.Component {
       .then(({data}) => {
         console.log("Axios", data)
```

Comment:
I think it would be ok to remove stray `console.log`s even though this is in the client.